src/main.rs

```diff
@@ -237,120 +237,109 @@ fn format_duration(duration: Duration) -> Result<String> {
 // Main function
 fn main() {
     pretty_env_logger::init();
-
     // Variables
     let mut table = Table::new();
     let matches = App::new("rsfetch")
-                    .version("1.5.0")
-                    .about("\nMy info fetch tool for Linux. Fast (0.01s - 0.2s execution time) and somewhat(?) minimal.\nAll \"BOOL\" options default to \"true\" (with the exception of editor, window manager, and ip address), and \"SOURCE\" defaults to no.")
+                    .version("1.6.0")
+                    .about("\nMy info fetch tool for Linux. Fast (0.01s - 0.2s execution time) and somewhat(?) minimal.\nAll options are on (with the exception of package count, editor, window manager, and ip address). Music info is turned off by default.")
                     .arg(Arg::with_name("credits")
                         .long("credits")
                         .value_name(" ")
-                        .help("Links to those who helped make this, and thanks to others who've helped me with my struggles.")
+                        .help("Links to those who helped make this, and thanks to others who've helped me.")
                         .takes_value(false))
                     .arg(Arg::with_name("bold")
                         .short("b")
                         .long("bold")
                         .value_name("BOOL")
-                        .help("Turn bold for field titles on or off.")
+                        .help("Turn bold for field titles off.")
                         .takes_value(true))
                     .arg(Arg::with_name("borders")
                         .short("B")
                         .long("borders")
                         .value_name("BOOL")
-                        .help("Turn borders on or off.")
+                        .help("Turn borders off.")
```

Comment:
Why is `bold` different from the others?

Reply:
It should be bold by default

Reply:
`bool` and `borders` have `takes_value(true)`, unlike the others.

Reply:
Quick question. How would I do this in rust?

let "x" = "y" if "condition is met";

Reply:
`let bold = !matches.is_preset("no-bold");`

Reply:
May I ask how that works?

Reply:
In Rust, as in basically every programming language, things like `10`, `true`, `matches` and `matches.is_present("no-bold")` are expressions. `10` is a number`, `true` is a boolean`, `matches` is a variable of whatever type `clap` has for it, and `matches.is_present("no-bold")` is a function or method call that evaluates to a boolean.

The `!` operator takes a boolean and returns its negation. And `let var = expr` binds the value of the `expr` expression to the variable called `var`.

In the meanwhile, `if` looks like `if expr_1 { expr_2 } else { expr_3 }`, where `expr_1` needs to be a boolean, and `expr_2` and `expr_3` must have the same type.

So there's nothing magic. `if` and `let` do different things, but both take expressions.

Reply:
That really explained a lot. Thank you for all of that information. I wish I would asked people these questions a lot sooner, it would've made everything so much easier.

Reply:
I think you mentioned some C background on Reddit. I don't know how much time you want to invest in learning programming, but it might be worth taking a course or reading a book. It doesn't have to be about Rust, as these things are almost universal.

Reply:
To be honest, I've barely used C. I'm definitely picking up some programming books soon though. I've had my eyes on a C++ and a Rust book. I think one thing to note though, is that everything I know has been either self-taught or learned through little collaborations like this. I've never actually had any coding or scripting taught to me professionally.

By the way, I believe I've fixed everything. Is there a way to delete this pull request so I can make a new one? Or can I just push the new changes and it'll automatically show up since this is comparing 2 branches?

Reply:
I've been thinking about it. I wouldn't necessarily recommend starting with C++ (it's very complex). Rust is better, I guess, but others recommend Python. But maybe going through at least the first chapter or two of [SICP](https://web.mit.edu/alexmv/6.037/sicp.pdf) wouldn't be a bad idea. The language is so simple you could fit its syntax on a napkin, letting you focus on what you're actually trying to do.

You can commit on the same branch and push the changes.

Reply:
I have some experience in Python. Not much though. Just some basic things like a systray icon, some gtk stuff, etc.

Thank you for the link, I'll make sure to read through the pdf.

I'll run the code through rustfmt real quick, then push the changes I made.